src/pages/Admin/Venue/TemplateCard/TemplateCard.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import classNames from "classnames";
+
+import { HIDDEN_BURN_VENUE_TEMPLATES, Template } from "settings";
+
+import "./TemplateCard.scss";
+
+export interface TemplateCardProps {
+  selected: boolean;
+  template: Template;
+  onClick: () => void;
+  thumbnail?: string;
+}
+
+export const TemplateCard: React.FC<TemplateCardProps> = ({
+  template: { template, name, description },
+  onClick,
+  selected,
+  thumbnail,
+}) => {
+  const containerClass = classNames({
+    TemplateCard: true,
+    // To help with "...so that a savvy admin can still create said venues..." @see https://github.com/sparkletown/internal-sparkle-issues/issues/1075#issue-974665804
+    [`TemplateCard__${template}`]: template,
+    "TemplateCard--hidden": HIDDEN_BURN_VENUE_TEMPLATES.includes(template),
```

Comment:
This addition goes in the spirit of
>Make hidden: Animate Map, Large Auditorium and Screening Room. When hiding venue templates, hide with inline style="display:none;" so that a savvy admin can still create said venues -- we may want to build some kind of "Advanced" and/or "Beta" grouped template dropdown UI eventually. For now this is our P1 approach.

from https://github.com/sparkletown/internal-sparkle-issues/issues/1075#issue-974665804
while also trying to honor https://github.com/sparkletown/sparkle/pull/2011#discussion_r693658387 (not embedding styles in DOM elements).